clusterloader2/pkg/measurement/common/service_creation_latency.go

```diff
@@ -304,6 +307,12 @@ func (p *pingChecker) run() {
 			}
 			// TODO(#685): Make ping checks less communication heavy.
 			pod, err := execservice.GetPod()
+			if err != nil {
+				klog.Warningf("No exec service pod found in cluster.")
```

Comment:
Please include the original error in the message, e.g.
```
klog.Warningf("call to execservice.GetPod() ended with error: %v", err)
```

Reply:
done.